submodules/power_cloudconnection_create/README.md

```diff
@@ -1,38 +1,36 @@
 # Submodule power-cloudconnection-create
 
-This submodule is used to create Cloud Connections and attach the cloud connections to the Transit gateway.
+This submodule creates Cloud Connections and attaches the cloud connections to the Transit gateway.
```

Comment:
It would be worth linking out to relevant Cloud documentation on Cloud Connections

Reply:
there would be general documentation on cloud doc that describes this automation (and how to use this manually)

Reply:
I still think we should make it easier for people consuming directly from github.com